src/modules/ie-objects/ObjectDetailPage.tsx

```diff
@@ -975,23 +976,31 @@ export const ObjectDetailPage: FC<DefaultSeoInfo> = ({ title, description, image
 		}
 
 		if (isKeyUser) {
-			return KEY_USER_ACTION_SORT_MAP(canDownloadMetadata);
+			return KEY_USER_ACTION_SORT_MAP(canDownloadMetadata || canDownloadNewspaper);
 		}
 
 		if (isKiosk) {
 			return KIOSK_ACTION_SORT_MAP();
 		}
 
 		if (isMeemooAdmin) {
-			return MEEMOO_ADMIN_ACTION_SORT_MAP(canDownloadMetadata);
+			return MEEMOO_ADMIN_ACTION_SORT_MAP(canDownloadMetadata || canDownloadNewspaper);
 		}
 
 		if (isCPAdmin) {
-			return CP_ADMIN_ACTION_SORT_MAP(canDownloadMetadata);
+			return CP_ADMIN_ACTION_SORT_MAP(canDownloadMetadata || canDownloadNewspaper);
```

Comment:
i would rename hasAccessToVisitorSpaceOfObject to canExport inside the KEY_USER_ACTION_SORT_MAP, MEEMOO_ADMIN_ACTION_SORT_MAP, CP_ADMIN_ACTION_SORT_MAP